### PR TITLE
activate swap for all projects

### PR DESCRIPTION
**Pull request by CvH**
Currently we only build RPiX with swap support and all other platforms not.
This PR removes from the projects the swap options and activate swap for all devices at distribution options.

opinions ?
@codesnake @MilhouseVH @lrusak @vpeter4

### CODE REVIEW

File: distributions/LibreELEC/options [hunk 1]
```diff
@@ -158,13 +158,13 @@
...
 # build with swap support (yes / no)
-  SWAP_SUPPORT="no"
+  SWAP_SUPPORT="yes"
 
 # swap support enabled per default (yes / no)
   SWAP_ENABLED_DEFAULT="no"
 
 # swapfile size if SWAP_SUPPORT=yes in MB
-  SWAPFILESIZE="256"
+  SWAPFILESIZE="128"
```
Comment (lrusak) [inline distributions/LibreELEC/options]: No point in changing this, it's only relevant if building with swap enabled default.

**Comment by lrusak** [comment]
I'm fine with this :+1:

File: distributions/LibreELEC/options (hunk 1 above)
Comment (CvH) [inline distributions/LibreELEC/options]: We have currently everywhere 128mb in use thats why I changed it to the value we currently use. If we keep 256 we would change the current setting - of course this should be a safe value too.

**Comment by vpeter4** [comment]
Didn't used swap before but I assume it would not hurt anything.

**Comment by codesnake** [comment]
Looks good to me :+1:

**Pull request merged**